AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -371,6 +370,23 @@ private IContainer CreateIoCContainer(IServiceCollection services)
                 await sampleData.CreateAdminUser();
             }
         }
+
+        private static void CheckForHangfireDatabaseAndCreateIfItDoesNotExist()
+        {
+            using (var sqlConnection = new SqlConnection("Server=(localdb)\\MSSQLLocalDB;Integrated Security=true;MultipleActiveResultsets=true;"))

```

Comment:
In that case, I don't think this approach will work. We can't assume that the connection string specified here will work for all developers. For example, Linux of Mac devs will be connecting to a remote database,

Can you remind me why we can't create the Hangfire database tables in the regular AllReady database?